packages/block-library/src/term-description/edit.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	useBlockProps,
+	BlockControls,
+	AlignmentToolbar,
+} from '@wordpress/block-editor';
+
+export default function TermDescriptionEdit( {
+	attributes,
+	setAttributes,
+	mergedStyle,
+} ) {
+	const { textAlign } = attributes;
+	const blockProps = useBlockProps( {
+		className: classnames( {
+			[ `has-text-align-${ textAlign }` ]: textAlign,
+		} ),
+		style: mergedStyle,
+	} );
+	return (
+		<>
+			<BlockControls>
+				<AlignmentToolbar
+					value={ textAlign }
+					onChange={ ( nextAlign ) => {
+						setAttributes( { textAlign: nextAlign } );
+					} }
+				/>
+			</BlockControls>
+			<div { ...blockProps }>
+				<div className="wp-block-term-description__placeholder">
+					<span>
```

Comment:
Do we need this extra `div` and `span`?

Reply:
It's what we have for the post-content block, so for consistency I copy-pasted the implementation.
I agree we most likely don't need it, but I'd rather we change all placeholders at once, keeping them consistent...